python/vyos/kea.py

```diff
@@ -104,6 +104,14 @@ def kea_parse_options(config):
             }
         )
 
+    if 'capwap-access-controller' in config:
+        options.append(
+            {
+                'name': 'capwap-ac-v4',
+                'data': config['capwap-access-controller'],
```

Comment:
```suggestion
    if 'capwap_access_controller' in config:
        options.append(
            {
                'name': 'capwap-ac-v4',
                'data': config['capwap_access_controller'],
```

Dashes are mangled to underscores within conf scripts